pylintrc

```diff
@@ -70,6 +70,7 @@
 ignore = 
 persistent = yes
 load-plugins = edx_lint.pylint,pylint_django,pylint_celery
+django-settings-module=edxsearch.settings
```

Comment:
Right at the top of this file, it says "DO NOT EDIT" :)
Could you move these changes to `pylintrc_tweaks` instead?

Reply:
Looks like you moved the changed to `pylintrc_tweaks` ok, but didn't revert these changes to `pylintrc`?